### PR TITLE
Update enrollment-restrictions-set.md

### PR DESCRIPTION
**Pull request by danivv96**

### CODE REVIEW

**Comment by PRMerger4** [comment]
@danivv96 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/enrollment/enrollment-restrictions-set.md [hunk 1]
```diff
@@ -146,13 +146,16 @@ Intune also blocks personal devices using these enrollment methods:
...
-        
+             > The min/max range isn't applicable to Apple devices that enroll with the Device Enrollment Program, Apple School Manager, or the Apple Configurator app. Although Intune doesn't block ADE enrollments that use Company Portal to authenticate, not meeting OS requirements impacts registration because devices can't create the Azure AD device record used to evaluate Conditional Access policies. You can tell that this is the case if a device user receives an error message that says "Couldn't map device record with a user" after they sign in to Company Portal.
+             
+          > [!NOTE]
+          > Windows does not provide revision(.rev) number to Intune during enrollment. So, the format to use in Enrollent Restriction Policy is "major.minor.build.0".
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/enrollment-restrictions-set.md:157]:
```suggestion
          > Windows does not provide a revision (.rev) number to Intune during enrollment. So, the format to use in Enrollment Restriction Policy is `major.minor.build.0`. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @danivv96, kindly commit the suggested changes. Many thanks.

**Comment by PRMerger5** [comment]
@danivv96 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: memdocs/intune/enrollment/enrollment-restrictions-set.md [hunk 2]
```diff
@@ -146,13 +146,16 @@ Intune also blocks personal devices using these enrollment methods:
...
-        
+             > The min/max range isn't applicable to Apple devices that enroll with the Device Enrollment Program, Apple School Manager, or the Apple Configurator app. Although Intune doesn't block ADE enrollments that use Company Portal to authenticate, not meeting OS requirements impacts registration because devices can't create the Azure AD device record used to evaluate Conditional Access policies. You can tell that this is the case if a device user receives an error message that says "Couldn't map device record with a user" after they sign in to Company Portal.
+             
+          > [!NOTE]
+          > Windows does not provide a revision (.rev) number to Intune during enrollment. Therefore, the format to use in Enrollment Restriction Policy is `major.minor.build.0`.
```
Comment (JohanFreelancer9) [inline memdocs/intune/enrollment/enrollment-restrictions-set.md:157]:
```suggestion
          > Windows does not provide a revision (.rev) number to Intune during enrollment. Therefore, the format to use in Enrollment Restriction Policy is `major.minor.build.0`. 
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by lenewsad** [comment]
Hi @danivv96, thank you for your contributions. I submitted one of the changes (removing "only") in a new PR. I am verifying the other change (new note) with the product team.

**Comment by yogkumgit** [comment]
@lenewsad Kindly update Thanks

**Comment by lenewsad** [comment]
Hi @yogkumgit, Pinging some additional SMEs to see if we can close this out.

**Comment by yogkumgit** [comment]
@lenewsad Any update? Thanks

**Comment by lenewsad** [comment]
Hi @danivv96, I confirmed with the product team that the format should be major.minor.build.0 as proposed. Thid PR is outdated so I made the change in a new PR. It should be live by end of day 4/25, PST. Thanks again for your contribution! 

CC: @yogkumgit

**Pull request closed without merging**